blogposts/sofi-moves-fast-on-hundreds-of-microservices-with-sourcegraph.md

```diff
@@ -0,0 +1,22 @@
+---
+title: SoFi moves fast on hundreds of microservices with Sourcegraph
+author: Vanesa Ortiz
+publishDate: 2019-11-19T10:00-07:00
+tags: [
+  blog
+]
+slug: sofi-moves-fast-on-hundreds-of-microservices-with-sourcegraph
+heroImage: https://about.sourcegraph.com/case-studies/sofi-sourcegraph-case-study.png
+published: true
+description: "With Sourcegraph, SoFi can innovate and move quickly while keeping up with hundreds of microservices."
+---
+
+<p style="text-align: center">
+  <img src="https://localhost:9000/case-studies/sofi-sourcegraph-case-study-og-embed.png" />
+</p>
+
+SoFi helps its over 900,000 members achieve financial independence to realize their ambitions with products for borrowing, saving, spending, investing, and protecting their money. 
```

Comment:
```suggestion
SoFi empowers its 900,000+ members to achieve their financial goals and independence through borrowing, saving, spending, and investing.
```

I suggest removing the "protecting their money" part as it felt redundant.

Reply:
This line was specifically added by the the director of communications at SoFi, so I'm hesitant to change it.